app/models/user.rb

```diff
@@ -3,13 +3,20 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :invitable, :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable,
-         :confirmable, :trackable, :lockable
+         :confirmable, :trackable, :lockable, validate_on_invite: true
 
   ROLES = %w[volunteer coordinator social_worker admin].freeze
   REGISTERABLE_ROLES = %w[volunteer coordinator social_worker].freeze
   validates :role, inclusion: { in: ROLES, message: '%{value} is not a valid role' }
 
   has_one :address, as: :addressable
+  has_and_belongs_to_many :offices
+
+  validate :has_at_least_one_office
+
+  def has_at_least_one_office
```

Comment:
Take it or leave it. I didn't see a spec directly covering this method.

Reply:
Yeah, you're right. The line gets hit by a request spec but should have a unit test for this too. I'll add that and ask for re-review. Thanks for catching it!